src/widgets/heightandweight/heightandweight-detailed-summary.component.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect } from "react";
+import { match, useParams } from "react-router";
+import SummaryCard from "../cards/summary-card.component";
+import styles from "./heightandweight-detailed-summary.css";
+import dayjs from "dayjs";
+import { getDimenionsObservationsRestAPI } from "./heightandweight.resource";
+import { useCurrentPatient } from "@openmrs/esm-api";
+import { isEmpty } from "lodash-es";
+import {
+  convertToPounds,
+  convertToFeet,
+  convertoToInches
+} from "./heightandweight-helper";
+
+export function HeightAndWeightDetailedSummary(
+  props: HeightAndWeightDetailedSummaryProps
+) {
+  const [dimensions, setDimensions] = useState<any>({});
+  const [
+    isLoadingPatient,
+    patient,
+    patientUuid,
+    patientErr
+  ] = useCurrentPatient();
+  let params = useParams();
+
+  useEffect(() => {
+    getDimenionsObservationsRestAPI(patientUuid).subscribe(response => {
```

Comment:
do you have to call this `getDimenionsObservationsRestAPI`? why not just `getDimenionsObservations`?

Reply:
`getDimenionsObservation` is specific to fhir. once fhir is stable will switch back to it. migrated it to use REST Webservices for now.